Dockerfile

```diff
@@ -26,3 +23,6 @@ RUN apk --update add --virtual build-dependencies git && \
     find /usr/lib/node_modules/npm -name test -o -name .bin -type d | xargs rm -rf ; \
     rm -rf /usr/share/man /tmp/* /root/.npm /root/.node-gyp \
     /usr/lib/node_modules/npm/man /usr/lib/node_modules/npm/doc /usr/lib/node_modules/npm/html)
+
+ENTRYPOINT ["/tini", "--", "./bin/application"]
+CMD [ "server" ]
```

Comment:
It is mostly aesthetics - most of our other docker files (and ones I have read) have this at the end, as it is sort of logically how the image will execute, where above it is more about the actual build.

The way we currently build, while optimal for image size, also always busts the cache, so it doesn't really affect that.